cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -443,13 +456,17 @@ private class ServiceImpl(
         val at = consentActionImpl.actionType
         if (at == ActionType.ACCEPT_ALL || at == ActionType.REJECT_ALL) {
 
+            val getChoiceIncludeData = IncludeData.generateIncludeDataForGetChoice(
+                includeDataGppParam = spConfig.gppConfig?.toIncludeDataGppParam(),
+            )
+
             val getChoiceParamReq = GetChoiceParamReq(
                 choiceType = at.toChoiceTypeParam(),
                 accountId = spConfig.accountId.toLong(),
                 propertyId = spConfig.propertyId.toLong(),
                 env = env,
                 metadataArg = campaignManager.metaDataResp?.toChoiceMetaData()?.copy(gdpr = null),
-                includeData = IncludeData.generateIncludeDataForGetChoice(),
+                includeData = getChoiceIncludeData,
```

Comment:
Since we don't do anything with the includeData variables, I'd get back to inlining all these includeData assignments, like:
```kotlin
includeData = IncludeData.generateIncludeDataForGetChoice(
    includeDataGppParam = spConfig.gppConfig?.toIncludeDataGppParam(),
)
```

Reply:
```kotlin
val getChoiceParamReq = GetChoiceParamReq(
  ...
  includeData = IncludeData.generateIncludeDataForGetChoice(
      includeDataGppParam = spConfig.gppConfig?.toIncludeDataGppParam(),
  )
  ...
)
```

The above looks perfectly fine to me. Let's inline it then.